util/build_configs/cray-internal/README.md

```diff
@@ -47,6 +48,16 @@ new subdir.
     A Cray system management tool, this script sets the default Chapel version obtained
     when a user runs `module load chapel` without specifying a specific version.
 
+* setenv-\*-\*.bash:
+  Setenv scripts for Chapel Cray RPMs of various types. These setenv scripts are based on
+  the setenv-example-3 script in the parent directory, and their basic structure is the
+  same.  However, many details were added to successfully build a complete Chapel Cray
+  module, including Python-venv, chpldoc, mason, 192 runtime configs, etc.
```

Comment:
I wouldn't fixate on (i.e., put in the comment) the specific value 192 here, since it changes as we add and subtract configs.  For example, it will change as soon as we add comm=ofi to the builds.